scripts/config/vite.config.dev.js

```diff
@@ -41,16 +50,31 @@ const getViteConfigDev = port => ({
     },
   },
   plugins: [
+    https ? basicSsl() : null,
     react(),
     {
-      name: 'configure-response-headers',
-      configureServer: server => {
+      name: "configure-response-headers",
+      configureServer: (server) => {
         server.middlewares.use((_req, res, next) => {
-          res.setHeader('Access-Control-Allow-Private-Network', 'true');
+          res.setHeader("Access-Control-Allow-Private-Network", "true");
           next();
         });
-      }
-    }],
+      },
+    },
+    {
+      name: "dev-server-endpoints",
+      configureServer: (server) => {
+        server.middlewares.use("/blocks.config.json", (req, res) => {
+          const json = fs.readFileSync("./blocks.config.json");
+          sendJson(res, JSON.parse(json));
+        });
+
+        server.middlewares.use("/git.config.json", (req, res) => {
+          sendJson(res, parseGitConfig.sync());
+        });
```

Comment:
nice! I'm guessing this will now add the `configureServer` headers - any idea if we still need the `Access-Control-Allow-Origin` header?

Reply:
yes, since the `configure-response-headers` plugin appears first, these endpoints have `Access-Control-Allow-Private-Network: true`.

We do still need `Access-Control-Allow-Origin: *` to avoid a CORS error:
```
Access to fetch at 'https://localhost:4000/git.config.json' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
(Hm I guess we could do `no-cors` instead on the `blocks-platform` side. Also there is a `cors` option in the Vite config but it didn't work when I tried it, maybe it doesn't apply to handlers from plugins.)